Jellyfin.Data/Enums/VideoRangeType.cs

```diff
@@ -45,6 +45,27 @@ public enum VideoRangeType
     /// </summary>
     DOVIWithSDR,
 
+    /// <summary>
+    /// Dolby Vision with Enhancment Layer (Profile 7).
+    /// </summary>
+    DOVIWithEL,
+
+    /// <summary>
+    /// Dolby Vision and HDR10+ Metadata coexists.
+    /// </summary>
+    DOVIWithHDR10Plus,
+
+    /// <summary>
+    /// Dolby Vision with Enhancment Layer (Profile 7) and HDR10+ Metadata coexists.
+    /// </summary>
+    DOVIWithELHDR10Plus,
+
+    /// <summary>
+    /// Dolby Vision with invalid configuration. e.g. Profile 8 compat id 6.
+    /// When using this range, the server would assume the video is still HDR10 after removing the Dolby Vision metadata.
+    /// </summary>
+    DOVIInvalid,
+
```

Comment:
Adding new enum members to an enum used in response model with no nullability or default value, like this one, is a breaking API change.
I'd like to avoid this for 10.11 so our clients can keep compatibility with both 10.10 and 10.11 as there is a significant database refactor.

If we want to avoid this in the future we should add a new "Unknown" member and use that as default value, or make the usages of this enum nullable.

Reply:
Except it does have a value "Unknown" already?

Reply:
Talking from an API perspective so it is somewhat different. The generated OpenAPI specification does not have a default value set for `MediaStream.videoRangeType`. To make that work we need to assign a default value or use the `[DefaultValue]` attribute on that property.

Right now the API specification is telling us "MediaStream.videoRangeType` is not null and has no default value. Therefor the SDK's are unable to fallback to a safe value if it receives an "unknown" value. I could make changes to our Kotlin SDK to always use "unknown" as default if that member exists but other OpenAPI code generators won't be able to figure out that behavior.

Reply:
I marked `VideoRangeType.Unknown` explicitly as default on api models using this enum. The more problematic thing is I *think* most if not all enums being used in API models that is not nullable already has its own implicit default value but not exposed to API yet. Is there any easy way to get all such APIs so that we can fix those at once?

Reply:
Model properties that are non-nullable using an enum that has an Unknown member in 10.10:

```
BaseItemDto.mediaType
BaseItemPerson.type
MediaSegmentDto.type
MediaStream.videoRange
MediaStream.videoRangeType
SearchHint.mediaType
```